Source/SwiftyCamViewController.swift

```diff
@@ -158,7 +161,11 @@ import AVFoundation
     public var allowAutoRotate                = false
 
     /// Specifies the [videoGravity](https://developer.apple.com/reference/avfoundation/avcapturevideopreviewlayer/1386708-videogravity) for the preview layer.
-    public var videoGravity                   : SwiftyCamVideoGravity = .resizeAspect
+    public var videoGravity                   : SwiftyCamVideoGravity = .resizeAspect {
+        didSet {
+            previewLayer.gravity = videoGravity
```

Comment:
Hi,
This line broke my app ( Nativescript camera-plus plugin use this branch ),
getting:
Fatal error: Unexpectedly found nil while unwrapping an Optional value: file SwiftyCam/SwiftyCamViewController.swift, line 166

Reply:
Missing a `?`